navigation/AppNavigator.js

```diff
@@ -1,10 +1,16 @@
-import React from 'react';
```

Comment:
Was this supposed to be taken out? What is the point of keeping it or removing this